mail_tracking/data/mail_message_subtype_data.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <record id="mail_message_subtype_failed_retry" model="mail.message.subtype">
```

Comment:
Why do we need a new subtype?

Reply:
To avoid retry to all followers

Reply:
But can we do it other way? I don't like too much to need a new subtype. And what happen on multiple failed messages in same thread? Are they notified again although they are not failed this time?

Reply:
What about this?

Reply:
Removed the message subtype... Added another hack to the collection: https://github.com/OCA/social/pull/411/files?file-filters%5B%5D=.py#diff-4e1cf268ea32e1266c1e39d18db0d429R207